src/solver/simulation/include/antares/solver/simulation/economy.h

```diff
@@ -98,8 +98,7 @@ class Economy
     uint pStartTime;
     uint pNbMaxPerformedYearsInParallel;
     std::vector<PROBLEME_HEBDO> pProblemesHebdo;
-    std::vector<std::unique_ptr<Antares::Solver::Optimization::WeeklyOptimization>>
-      weeklyOptProblems_;
+    std::vector<std::unique_ptr<Optimization::WeeklyOptimization>> weeklyOptProblems_;
```

Comment:
This `unique_ptr` is no longer necessary.
```suggestion
    std::vector<Optimization::WeeklyOptimization> weeklyOptProblems_;
```